src/parser/core/Combatant.ts

```diff
@@ -279,6 +280,14 @@ class Combatant extends Entity {
     return this.conduitsByConduitID[spellId] && this.conduitsByConduitID[spellId].rank;
   }
 
+  likelyHasEmpoweredConduits() {
```

Comment:
I think it'd be worth to call this function in the conduitRankBySpellID function and return rank + 2 if true, otherwise just rank. That ensures that the individual contributor has to do as little as possible to support empowered conduits.